src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx

```diff
@@ -0,0 +1,152 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent } from 'parser/core/Events';
+import DonutChart from 'parser/ui/DonutChart';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { SPELL_COLORS } from '../../constants';
+import { getHealForLifebindHeal } from '../../normalizers/CastLinkNormalizer';
+
+class Lifebind extends Analyzer {
+  healingBySpell: Map<number, number> = new Map<number, number>();
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS_EVOKER.LIFEBIND_TALENT);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.LIFEBIND_HEAL), this.onHeal);
+  }
+
+  onHeal(event: HealEvent) {
+    const triggerSpell = getHealForLifebindHeal(event);
+    const spellID = triggerSpell ? triggerSpell.ability.guid : 0;
+    this.healingBySpell.set(
+      spellID,
+      (this.healingBySpell.get(spellID) ?? 0) + event.amount + (event.absorbed || 0),
+    );
+  }
+
+  healingForSpell(spellId: number): number {
+    return this.healingBySpell.get(spellId) ?? 0;
+  }
+
+  renderDonutChart() {
```

Comment:
Not sure if intended or not but this doesn't chart doesn't have off of the spells from your core healing spells obj.
Missing: 
Rewind, Dream Flight, Cycle of Life, Fluttering Seedlings, Echo,

Reply:
Yeah you're right, I think I started adding it and didn't finish it. Let me add the rest.